### PR TITLE
Add cluster.name and host.ems_ref to manifest

### PR DESCRIPTION
**Pull request by agrare**

### CODE REVIEW

File: config/cfme/manifest_5_11.json [hunk 1]
```diff
@@ -142,6 +142,7 @@
       },
       "ems_clusters": {
         "id": null,
+	"name": null,
```
Comment (bdunne) [inline config/cfme/manifest_5_11.json]: The alignment looks to be off just a little bit

File: config/cfme/manifest_5_11.json [hunk 2]
```diff
@@ -266,6 +271,7 @@
...
```
Comment (bdunne) [inline config/cfme/manifest_5_11.json]: same here

File: config/cfme/manifest_5_11.json (hunk 1 above)
Comment (agrare) [reply config/cfme/manifest_5_11.json]: Hm you're right I think a tab made its way in there

Comment (agrare) [reply config/cfme/manifest_5_11.json]: Good catch, fixed

**Pull request merged**